src/App/Pages/Vault/AddEditPageViewModel.cs

```diff
@@ -835,27 +836,44 @@ private void TriggerCipherChanged()
             TriggerPropertyChanged(nameof(Cipher), _additionalCipherProperties);
         }
 
-        private async void CheckPasswordAsync()
+        private async Task CheckPasswordAsync()
         {
-            if (!(Page as BaseContentPage).DoOnce())
-            {
-                return;
-            }
             if (string.IsNullOrWhiteSpace(Cipher.Login?.Password))
             {
                 return;
             }
+
             await _deviceActionService.ShowLoadingAsync(AppResources.CheckingPassword);
-            var matches = await _auditService.PasswordLeakedAsync(Cipher.Login.Password);
-            await _deviceActionService.HideLoadingAsync();
-            if (matches > 0)
+            try
             {
-                await _platformUtilsService.ShowDialogAsync(string.Format(AppResources.PasswordExposed,
-                    matches.ToString("N0")));
+                var matches = await _auditService.PasswordLeakedAsync(Cipher.Login.Password);
+                await _deviceActionService.HideLoadingAsync();
+                if (matches > 0)
+                {
+                    await _platformUtilsService.ShowDialogAsync(string.Format(AppResources.PasswordExposed,
+                        matches.ToString("N0")));
+                }
+                else
+                {
+                    await _platformUtilsService.ShowDialogAsync(AppResources.PasswordSafe);
+                }
             }
-            else
+            catch (ApiException apiException)
             {
-                await _platformUtilsService.ShowDialogAsync(AppResources.PasswordSafe);
+                _logger.Exception(apiException);
+                await _deviceActionService.HideLoadingAsync();
+                apiException.Error = new Core.Models.Response.ErrorResponse();
```

Comment:
why are you creating a new Error here? this shouldn't be needed

Reply:
This was for testing purposes. Removed from ViewPageViewModel and forgot to remove from here. Nice catch thanks